chapters/3-measurement.tex

```diff
@@ -172,32 +258,38 @@ \subsection{Defining study comparisons using master data}
 Instead, the treatment group is compared to a control group
 that is statistically indistinguishable,
 which makes the average impact of the treatment
-mathematically equivalent to the difference in averages between the groups.
+mathematically equivalent to 
+the difference in averages between the groups.
 Statistical similarity is often defined
 as \textbf{balance} between two or more groups.	
 Since balance tests are commonly run for impact evaluations,
 DIME Analytics created a Stata command to 
 standardize and automate the creation of nicely-formatted balance tables:
 \texttt{iebaltab}\sidenote{
 	\url{https://dimewiki.worldbank.org/iebaltab}}.
-Each research design has a different method	for identifying the statistically-similar control group. 
-for how the statistically similar control group is identified.
 
-The rest of this section covers how data requirements differ
-between different research designs.
-What does not differ, however, is that the authoritative source
-for which units are in the treatment group and which are in the control group
-should always be one or several variables in your master dataset. 
-You will often have to merge that data to other datasets, 
-but that is an easy task if you created a data map.
+Each research design has a different method for
+identifying the statistically-similar control group. 
+The rest of this section covers how research data requirements 
+differ between those different methods.
+What does not differ, however, 
+is that these data requirements are all research variables.
+The source for the required research variables varies
+between research designs and between projects, 
+but the authoritative source for that type of data should
```

Comment:
```suggestion
is that these data requirements are all research variables.
is that the research variables generated in the processes described below should always be included in the master dataset.
```

Reply:
I updated the suggestion to say "discussed below" instead of "generated below", as in this section we do not yet talk about how to generate them.

Reply:
done d305ab2c71522ecaa20290059ca31a5ea0bbfca2